lib/presentation/widget/upload_request_group/upload_request_group_viewmodel.dart

```diff
@@ -47,10 +49,22 @@ import 'package:redux_thunk/redux_thunk.dart';
 class UploadRequestGroupViewModel extends BaseViewModel {
   final AppNavigation _appNavigation;
   final GetAllUploadRequestGroupsInteractor _getAllUploadRequestGroupsInteractor;
+  final GetSorterInteractor _getSorterInteractor;
```

Comment:
Missing implementations for `_getSorterInteractor`

Reply:
Yep forget it thx